cmd/moby/trust_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
```

Comment:
I dont really like adding another vendor just for a test, surely the standard library is not that much pain? You can make an array and just test all of it...